packages/atomic/cypress/integration/facet-standard.cypress.ts

```diff
@@ -0,0 +1,406 @@
+import {getAnalyticsAt, getApiRequestBodyAt} from '../utils/network';
+import {setUpPage, shouldRenderErrorComponent} from '../utils/setupComponent';
+import {
+  FacetSelectors,
+  createAliasShadow,
+  createAliasFacetUL,
+  createBreadcrumbShadowAlias,
+} from './facet-selectors';
+
+const facetProp = {
+  field: 'author',
+  label: 'Authors',
+};
+
+const compareAlphanumericalValues = (valueA: string, valueB: string) => {
```

Comment:
You should use localeCompare: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare